CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```

Comment:
You can remove the second please word, like this:

"Please note we have a code of conduct, follow it in all your interactions with the proejct."